docs/examples/rust/secure_message_client_verify.rs

```diff
@@ -47,7 +47,7 @@ fn main() {
     let public_key = PublicKey::try_from_slice(public_key).expect("parse public key");
 
     let socket = UdpSocket::bind("localhost:0").expect("client socket");
-    socket.connect(&remote_addr).expect("client connection");
+    socket.connect(remote_addr).expect("client connection");
```

Comment:
With these being gone, you can also remove the corresponding `#[allow(...)]` suppressions in lib.rs files.

Reply:
It is my first look into Rust with Themis. I want to begin a new pull request and add the little pieces of patches—for example, a patch for Ubuntu 20.04, a patch for Obj-C, etc. 
One question: I could test the Rust code without errors when I installed Rust toolchain 1.56 on my computer. Why CI produces the error? Do you have ideas?

Reply:
> Why CI produces the error? Do you have ideas?

Because it checks stable version as well vOv

Most warnings come from newer Clippy, so 1.56 version is totally happy with the code as is. While stable requires all those fixups that you made, correcting warnings it produces.

Since your plan is to fix all the warnings and make a patch release, you'd better sync up CI configuration between `master` and `release/0.14` so that both test Rust with stable and 1.56.

Reply:
Or you can fix warnings only on `master` and just ignore them here, since they are not critical.

Reply:
+1 to ignore warnings here